pkg/deployments/tasks/20220413-child-chain-gauge-factory/input.ts

```diff
@@ -8,24 +8,21 @@ export type ChildChainLiquidityGaugeFactoryDeployment = {
 
 const AuthorizerAdaptor = new Task('20220325-authorizer-adaptor', TaskMode.READ_ONLY);
 const Vault = new Task('20210418-vault', TaskMode.READ_ONLY);
-const TestBalancerToken = new Task('20220325-test-balancer-token', TaskMode.READ_ONLY);
+const Tokens = new Task('00000000-tokens', TaskMode.READ_ONLY);
 
 export default {
   AuthorizerAdaptor,
   Vault,
```

Comment:
```suggestion
const BAL = new Task('00000000-tokens', TaskMode.READ_ONLY);

export default {
  AuthorizerAdaptor,
  Vault,
  BAL
```

This avoids us having to break out the different network configs.